src/components/StaticContent/FAQs.re

```diff
@@ -75,9 +75,9 @@ module FAQs = {
                   )}>
                   <div>
                     <div className=faqExplainerText>
-                      <Text>
+                      <Text className=Styles.colorGreen>
```

Comment:
This green text looks bad everywhere...

Reply:
I think it was a mistake to leave it in

Reply:
Orginally both texts were black (Q and A), and it was difficult to distinguish between the two. I think this is better than the original, although @DenhamPreen is going to style the FAQs at somepoint I think, which will make this irrelevant.

Reply:
I think rather than green we need to change the font size of the heading and text and add a line break below the explanation. Will add to the trello board for me.